src/renderer/store/modules/plugin.js

```diff
@@ -44,7 +44,9 @@ export default {
 
         let match = true
 
-        if (category && category !== 'all') {
+        if (category === 'all') {
+          match = match && !plugin.config.categories.includes('theme')
+        } else if (category && category !== 'all') {
```

Comment:
```diff
- } else if (category && category !== 'all') {
+ } else if (category) {
```
Too late but here but the check against `'all'` can be removed.

Reply:
yes

Reply:
where were you? we needed you